projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/datajobs/MockKubernetes.java

```diff
@@ -44,50 +44,23 @@ public class MockKubernetes {
   public DataJobsKubernetesService mockDataJobsKubernetesService()
       throws ApiException, IOException, InterruptedException {
     DataJobsKubernetesService mock = mock(DataJobsKubernetesService.class);
-    mockKubernetesService(mock);
-    return mock;
-  }
 
-  @Bean
```

Comment:
The diff of this file is very bad in github and should probably be looked at in your ide.